### ISSUE TITLE
Can you add a portable version to the releases?

### ISSUE THREAD

**Issue by MikeCarlo** [pbidax/DAXDebugOutput#1]
I would like to integrate this tool as an external tool within my Distribution product for external tools [Business Ops](https://docs.microsoft.com/en-us/power-bi/transform-model/desktop-external-tools#featured-open-source-tools) 

The easiest method to integrate is using a portable version of the software and not an MSI file. Would you be able to add a portable version?

**Comment by JamesDBartlett3** [comment]
Yes, please! In addition to making DAXDebugOutput more easily integrated with other tools, having a portable version would also allow people who don't have admin rights (and thus cannot install software) to just run the app directly.

**Comment by somedaygone** [comment]
Because Microsoft doesn't have a directory or "AppSource" location to publish External Tools, Business Ops is the de facto way to find new External Tools. A lot more people would know DAX Debug exists if it could be added to Business Ops.

**Comment by SQLGuyJPS** [comment]
Ditto all the above. Please add this capability!

**Comment by JamesDBartlett3** [comment]
@pbidax Are there any plans to add a portable version of this tool, so that we can add it into Business Ops?
